### DISCUSSION TITLE
How to authenticate an app

### DISCUSSION THREAD

**Discussion by masmc05** [plan-player-analytics/Plan#3719] [APIs and Development]
Hello, would like to access endpoints like https://example.com/v1/player?player=masmc05 externally, but idk how to authenticate the app to access it, as i've not seen anywhere some api tokens etc. How can I do that?

**Comment by AuroraLS3** [comment]
Use /auth/login endpoint and pass the cookie from Set-Cookie header from the response to Cookie header of further requests.

The /docs page on the plugin webserver should have info on how to use the endpoint (uri encoded user=user&pass=pass in the request body)

**Comment by masmc05** [comment]
Thanks
